client/app/page.tsx

```diff
@@ -1,20 +1,41 @@
 'use client'
-import { Button, Heading } from '@/_components/ui'
-import { useGetMeQuery } from '@/_hooks/api'
+
+import { Alert as NoHouseAlert, TodosList } from './(todos)/components'
+import { useGetTodosQuery } from './(todos)/hooks'
+import { Loading } from './_components/layout'
+import { Button, Heading } from './_components/ui'
 
 export default function Home() {
-  // This is the sample. Please change it to todos API query.
-  const { data } = useGetMeQuery()
+  const { data: todos, isError, isLoading } = useGetTodosQuery()
+
+  if (isLoading) {
+    return <Loading isCenter />
+  }
+
+  // TODO: Replace this with the error component
+  if (isError) {
+    return <p>Failed to load data.</p>
+  }
+
+  if (!todos) {
+    return (
+      <NoHouseAlert
+        mainMessage='You are currently not in any house.'
+        // TODO: Replace the linkHref with the actual path
+        linkHref='/create-setting'
+        linkText='Create house'
+        additionalText='or wait for an invitation'
+      />
+    )
+  }
 
   return (
-    <div className='px-4 pb-10 pt-8 md:px-14 md:pb-20 md:pt-10'>
+    <div className='flex flex-col gap-8 px-4 pb-10 pt-8 text-slate-800 md:px-14 md:pb-20 md:pt-10 '>
       <Heading
-        title='Fairy share'
+        title={todos.houseName}
```

Comment:
Can you also change the rules button to redirect to `/rules`?

Reply:
Fixed it!
https://github.com/HAK2024/fairy-share/pull/71/commits/4b1deb9c93b0fc0758b4f0284dc44290cb814eec